tests/e2e/util.go

```diff
@@ -483,6 +483,49 @@ type GetTaskTstatus struct {
 	OrgID           string      `json:"org_id"`
 }
 
+type TanzuCluster struct {
```

Comment:
add a comment to describe about the struct

Reply:
done